nest/src/channel/channel.service.ts

```diff
@@ -420,9 +426,9 @@ export class ChannelService {
       }
       /* Hash the password */
       if (typeof dto.passwordHash === 'string' && dto.type === 'PROTECTED') {
-        dto.passwordHash = await argon.hash(dto.passwordHash, {
-          type: argon.argon2id,
-        });
+        dto.passwordHash = dto.passwordHash; // await argon.hash(dto.passwordHash, {
```

Comment:
why did you remove the hashing here again ?

Reply:
I am having issues with argon, it just crashes the backend. I is definetely an issue on my end but I am not sure how to fix it, so to work I have to comment out all the lines that use argon.

Reply:
Fair enough maybe @williamollio can take a look if he have time or we can take a look together on thursday

Reply:
Sure